### DISCUSSION TITLE
DataContract catalog

### DISCUSSION THREAD

**Discussion by belaboros** [datamesh-architecture/datacontract-specification#82] [Q&A] [answered]
Hi All, 

What tools do you suggest to expose data contracts in a human readable form in a catalog e.g.: for users of a Databricks environment?

**Comment by simonharrer** [comment, marked as answer]
You can either create a HTML read-only catalog using the `catalog` command of the [Data Contract CLI](https://cli.datacontract.com). An example of such a generated catalog is available at https://datacontract.com/examples/

If you need a more sopisticated solution, have a look at the [Data Contract Manager](https://datacontract-manager.com). You can even publish your data contracts to the Data Contract Manager via the `publish` command of the Data Contract CLI.

Hope this helps.
